testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/web/ListenerTestCase.java

```diff
@@ -309,8 +405,10 @@ private ModelNode getRemoveConnectorOp(Listener conn) {
     static class SecurityRealmsSetup extends AbstractSecurityRealmsServerSetupTask {
         @Override
         protected SecurityRealm[] getSecurityRealms() throws Exception {
+            setUpKeyStores();
             URL keystoreResource = Thread.currentThread().getContextClassLoader().getResource("security/server.keystore");
             URL truststoreResource = Thread.currentThread().getContextClassLoader().getResource("security/jsse.keystore");
+            ClassLoader clloader = Thread.currentThread().getContextClassLoader();
```

Comment:
Is this needed? It doesn't look like it's referenced anywhere.

Reply:
Fixed